### ISSUE TITLE
Get test coverage up 

### ISSUE THREAD

**Issue by ptoffy** [vapor/jwt-kit#138]

**Comment by adam-rocska** [comment]
@ptoffy Are contribs welcome? I'm relying on jwt-kit and was thinking of getting my hands wet somehow.

**Comment by ptoffy** [comment]
@adam-rocska of course! Feel free to ask here or on the Vapor Discord if you have any questions
